notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2515,6 +2515,59 @@ mother and son [Pluto Nash's mother Flura informed Pluto that Rex Crater had ass
 celebrity worship [The small time smuggler Felix Laranga was a big fan of Pluto Nash. He had Pluto sign his helmet, and blew Pluto's cover at the hotel/casino by informing everyone within earshot that Pluto was in the room.]
 
 
+movie: Simone (2002)
+====================
+
+:: Title
+Simone
+
+:: Description
+Simone (stylized as S1m0ne) is a 2002 American science fiction film written,
+produced, and directed by Andrew Niccol. It stars Al Pacino, Catherine Keener,
+Evan Rachel Wood, Rachel Roberts, Jay Mohr, and Winona Ryder.
+
+Synopsis: A fading director creates a virtual actress to star in his films and
+attempts to keep her non-human nature a secret as she becomes more and more
+famous.
+
+:: Date
+2002-08-23
+
+:: References
+https://en.wikipedia.org/wiki/Simone_(2002_film)
+
+:: Ratings
+1 <paul>
```

Comment:
An outrage. It's Al Pacino! Al Pacino!

Reply:
A chef is only as good as his ingredients.